fastpair/rust/src/bluetooth/common/device.rs

```diff
@@ -21,7 +21,7 @@ use crate::bluetooth::common::ServiceData;
 /// They provide methods for retrieving device info and running device actions,
 /// such as pairing.
 #[async_trait]
-pub trait Device: Sized {
+pub trait Device: Sized + Send + Sync {
```

Comment:
correct me if i'm wrong here but this seems... strange.

Reply:
@marshallpierce

Reply:
Hard to say... You'd need this if Device is intended to be shared across threads, but hard to say in isolation if that's actually reasonable.